sdk/src/main/java/com/kin/ecosystem/widget/KinEcosystemTextView.kt

```diff
@@ -0,0 +1,40 @@
+package com.kin.ecosystem.widget
+
+import android.content.Context
+import android.graphics.Typeface
+import android.util.AttributeSet
+import android.widget.TextView
+import com.kin.ecosystem.R
+import com.kin.ecosystem.obtainAttrs
+import com.kin.ecosystem.widget.util.FontUtil
+
+class KinEcosystemTextView @JvmOverloads constructor(context: Context,
+                                                     attrs: AttributeSet? = null,
+                                                     defStyleAttr: Int = 0) : TextView(context, attrs, defStyleAttr) {
+
+    init {
+        val attributes = obtainAttrs(attrs, R.styleable.KinEcosystemTextView)
+
+        try {
+            attributes?.let {
+                val fontExtra = it.getInt(R.styleable.KinEcosystemTextView_fontExtra, REGULAR)
+                typeface = getTypeFace(fontExtra)
+            }
+        } finally {
+            attributes?.recycle()
+        }
+    }
+
+    private fun getTypeFace(fontStyle: Int): Typeface {
+        return when(fontStyle) {
+            REGULAR -> FontUtil.SAILEC
```

Comment:
can use only one case of meduim and defualt is the regular - what ever you prefer